src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -2622,7 +2629,7 @@ protected function getArrayableItems(array $values)
      */
     public function getAttribute($key)
     {
-        if (array_key_exists($key, $this->attributes) || $this->hasGetMutator($key)) {
+        if (array_key_exists($key, $this->attributes) || $this->hasGetMutator($key) || $this->isAlias($key)) {
```

Comment:
`protected function hasAlias()` is not used by any other method, do we really need it?